hpedockerplugin/hpe/hpe_3par_common.py

```diff
@@ -464,6 +464,72 @@ def create_vlun(self, volume, host, is_snap, nsp=None, lun_id=None):
                               vlun_info['lun_id'],
                               nsp)
 
+    def create_snap_schedule(self, src_vol_name, schedName, snapPrefix,
```

Comment:
I Agree with your point Imran.  Will Handle it as separate activity, so that all mount_conflict_delay(remove vlun), create schedule and remove schedule can be handled in that class.

Reply:
No problem Swapnil. We will take that up as a separate activity.